api_tests/users/views/test_user_detail.py

```diff
@@ -95,6 +108,16 @@ def test_get(self, app, user_one, user_two):
         user_json = res.json['data']
         assert 'profile_image' in user_json['links']
 
+    #   user_viewed_through_anonymous_link
+        url = '/{}users/{}/?view_only={}'.format(API_BASE, user_one._id, view_only_link.key)
```

Comment:
Ah, right, we are *super* aggressive with VOLs in the API. I've warned Abram about this behavior.